docs/examples/lambdaway_eval.js

```diff
@@ -1082,10 +1082,10 @@ var toggle_visibility = function ( id ) {
   getId(id).style.visibility = (OK)? "hidden" : "visible";
 }; // toggle_visibility
 var doSave = function () {
-  return confirm( "Save and publish modifs ?" );
+  return confirm( "Save and publish changes ?" );
```

Comment:
`modifications` would be the correct word, but I went for `changes` since it's closest by character count.

This could be an indication of quirkiness, but I didn't read it that way.